docs/DeveloperGuide.md

```diff
@@ -151,8 +151,31 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 --------------------------------------------------------------------------------------------------------------------
 ## **WellNus Implementation**
 
+### Student Notes feature
 
+The adding of student notes is facilitated by `NoteCommand`. It extends `Command` and allows the addition of a `Note`
+to the student at the specified index. The `Note` given in the user input can contain any
```

Comment:
idk if this should be in the DG, cause seems more for like user guide uh

Reply:
good point, removed the less relevant portion